package.json

```diff
@@ -1,5 +1,495 @@
 {
```

Comment:
I don't know a lot about package.json in general but 2 thoughts - (a) this looks a bit huge, and possibly maintenance issue - what happens when we get later version. of cypress? I guess just update this file (b) can we use package.json to install any other things while we're at it. Are there any other npm packages it would be have installed.

Reply:
a) this lot was auto installed when doing the `npm i --saveDev` for the dependancies. so should be ok, package manager will handle this. 
b) absolutely! but as and when we need it.